client/packages/common/src/ui/layout/tables/columns/VaccineVialManagementStatusColumn.tsx

```diff
@@ -0,0 +1,37 @@
+import { TypedTFunction, LocaleKey } from '@common/intl';
+import { RecordWithId } from '@common/types';
+import { UNDEFINED_STRING_VALUE } from '@common/utils';
+import { ColumnDefinition, ColumnAlign } from './types';
+import { StockOutLineFragment } from 'packages/invoices/src/StockOut';
+
+export const getVaccineVialManagementStatusColumn = <
```

Comment:
Would just rename this to Vvm to match the rest of the codebase and refactor for line edit please~